python/cugraph/cugraph/testing/__init__.py

```diff
@@ -11,6 +11,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cugraph.testing.utils import (
-    RAPIDS_DATASET_ROOT_DIR_PATH,
+from cugraph.testing.utils import RAPIDS_DATASET_ROOT_DIR_PATH, RAPIDS_DATASET_ROOT_DIR
+
+#
+# Moved Dataset Batches
+#
+
+# FIXME: it is hard to keep track of which datasets are included in certain batches and which are not. maybe convert this to a single fixture that let's you import datasets by name and yields a list? eg. get_batch("A", "B", C") => [A, B, C]
+# batches
+
+from cugraph.datasets import (
+    cyber,
+    dolphins,
+    karate,
+    ktruss_polbooks,
+    polbooks,
+    netscience,
+    small_line,
+    small_tree,
+    email_Eu_core,
+    toy_graph,
+    toy_graph_undirected,
 )
+
+DATASETS_UNDIRECTED = [karate, dolphins]
+DATASETS_UNDIRECTED_WEIGHTS = [netscience]
+DATASETS_SMALL = [karate, dolphins, polbooks]
+STRONGDATASETS = [dolphins, netscience, email_Eu_core]
+DATASETS_KTRUSS = [(polbooks, ktruss_polbooks)]
+MEDIUM_DATASETS = [polbooks]
+SMALL_DATASETS = [karate, dolphins, netscience]
+RLY_SMALL_DATASETS = [small_line, small_tree]
+ALL_DATASETS = [
+    dolphins,
+    karate,
+    ktruss_polbooks,
+    polbooks,
+    netscience,
+    small_line,
+    small_tree,
+    email_Eu_core,
+    toy_graph,
+    toy_graph_undirected,
+]
+ALL_DATASETS_WGT = [karate, dolphins, netscience, polbooks, small_line, small_tree]
+TEST_GROUP = [dolphins, netscience]
+
+# FIXME: removed karate variant. check if unit tests are breaking
+DATASETS_UNRENUMBERED = []
+DATASETS = [dolphins, netscience, karate]
```

Comment:
This is a lot better, but we should still clean these up if possible: eg. `polbook` is in both the "small" and "medium" lists, we use underscores most of the time but `STRONGDATASETS` does not, `RLY` is an abbreviation but we don't abbreviate anything else, `DATASETS_KTRUSS` should be defined in the ktruss tests if possible instead of here (plus it's using a tuple and no others are), etc.  I'm also still thinking we may not want the ALL_CAPS naming convention here, unless anyone else thinks it's helpful in some way.

Reply:
I've taken a closer look at the unit tests and figuring out just how many of these batches actually need to be supported. It looks like a handful of them can be removed, since they were leftover namespaces from previous iterations of our tests. The batches that are not being used more than once can also be declared within the scope of the test file and be removed from `testing/__init__.py`

It looks like the following can be safely removed:
- `TEST_GROUP`: no longer used
- `DATASETS_UNRENUMBERED`: can be replaced with `karate_disjoint`
- `DATASETS_UNDIRECTED_WEIGHTS`: can be replaced with `netscience`
- `STRONGDATASETS`: no longer used
- `RLY_SMALL_DATASETS`: no longer used
- `MEDIUM_DATASETS`: can be replaced with `polbooks`
- `DATASETS_KTRUSS`: can be declared inside `test_k_truss.py`